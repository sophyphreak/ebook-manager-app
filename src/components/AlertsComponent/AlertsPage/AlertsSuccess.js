import React from 'react';
import SuccessPage from '../../SuccessPage/SuccessPage';

const AlertsSuccess = (props) => (
  <SuccessPage {...props} />
);

export default AlertsSuccess;