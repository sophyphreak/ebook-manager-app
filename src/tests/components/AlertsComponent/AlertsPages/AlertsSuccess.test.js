import React from 'react';
import { shallow } from 'enzyme';
import AlertsSuccess from '../../../../components/AlertsComponent/AlertsPage/AlertsSuccess';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AlertsSuccess />);
});

test('should render AlertsSuccess correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
