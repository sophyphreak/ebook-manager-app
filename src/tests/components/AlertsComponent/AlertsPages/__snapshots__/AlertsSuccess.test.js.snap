// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AlertsSuccess correctly 1`] = `<SuccessPage />`;
